lemur/plugins/lemur_aws/elb.py

```diff
@@ -87,6 +87,48 @@ def get_all_elbs(**kwargs):
         raise
 
 
+def _filter_ignored_elbsv1(elbs, **kwargs):
+    return _filter_ignored_elbs(elbs, "LoadBalancerName", "LoadBalancerNames", "LoadBalancerName", **kwargs)
+
+
+def _filter_ignored_elbsv2(elbs, **kwargs):
+    return _filter_ignored_elbs(elbs, "LoadBalancerArn", "ResourceArns", "ResourceArn", **kwargs)
+
+
+def _filter_ignored_elbs(elbs, key_field, arg_name, response_key_field, **kwargs):
+    """
+    Look up tags and remove any ELBs that should be ignored.
+    :param elbs:
+    :param kwargs:
+    :return:
+    """
+    if not elbs:
+        return elbs
+    ignore_tag = current_app.config.get("AWS_ELB_IGNORE_TAG")
+    if not ignore_tag:
+        return elbs
+    try:
+        keys = [elb[key_field] for elb in elbs]
+        client = kwargs.pop("client")
+        tags_list = client.describe_tags(**{arg_name: keys})["TagDescriptions"]
+        ignored_keys = {}
+        for tags in tags_list:
```

Comment:
This is where I get a bit lost, due to all the formats AWS endpoint tags can have. 
If it is not much trouble, an example formatting of `tags` and `elb` would it easier to follow the code.

Reply:
Added some more docs and the response format, hope this helps.